DisplaySettings/DisplaySettings.cpp

```diff
@@ -3953,6 +3953,7 @@ namespace WPEFramework {
                     LOGERR("HdmiCecSink Initialisation failed\n");
                 }
                 else {
+                    WPEFramework::JSONRPC::LinkType<WPEFramework::Core::JSON::IElement>* client = (WPEFramework::JSONRPC::LinkType<WPEFramework::Core::JSON::IElement>*) m_client;
```

Comment:
Pls remove this and just use m_client. Also need to synchronize access to m_client since it's used/set in different threads.